### DISCUSSION TITLE
GLTF file not working

### DISCUSSION THREAD

**Discussion by rohan-nara** [google/filament#5779] [Q&A]
modelViewer.loadModelGltf(buffer) { uri -> readAsset("models/$uri") }  not Executing in load gltf function.

snippet:

       private fun loadGltf(name: String) {

        val buffer = readAsset("models/${name}.gltf")
        modelViewer.loadModelGltf(buffer) { uri -> readAsset("models/$uri") } // this function not executing
        Log.v("working","gltf function called") // LOG I ADDED
        modelViewer.transformToUnitCube()

    }

this is the code Snippet  from (https://medium.com/@philiprideout/getting-started-with-filament-on-android-d10b16f0ec67) .

THE PROBLEM I'M GETTING IS : 
WHEN I LOAD BUSTERDRONE.gltf FILE  ,   IT WORKS FINE , BUT
WHEN I LOAD MY .gltf FILE  , ALL I SEE IS A  BLACKSCREEN.

**Comment by bejado** [comment]
Are you able to load your glTF file in other viewers, such as https://gltf-viewer.donmccurdy.com/ ? Could you link your glTF file here?

**Reply by rohan-nara** [reply]
[arya-1.zip](https://github.com/google/filament/files/9098576/arya-1.zip)
![image](https://user-images.githubusercontent.com/64644909/178641198-336686aa-9730-4856-a9c8-1fbbfeedf9f2.png)
these are the warnings I get when i view in gltf viewer.

**Reply by bejado** [reply]
Does the model appear correctly with the online glTF viewer?

**Reply by rohan-nara** [reply]
yes
